addons/hr_holidays/tests/test_allocations.py

```diff
@@ -385,3 +386,27 @@ def test_automatic_allocation_type(self):
         })
 
         self.assertEqual(allocation.allocation_type, 'regular')
+
+    def test_create_allocation_from_company_with_no_employee_for_current_user(self):
+        """
+            This test makes sure that the allocation can be created if the current company doesn't have an employee
+            linked to the loggedIn user, the form initializes with default employee the one linked to the user
+            now it initializes with empty employee
```

Comment:
```suggestion
            This test makes sure that the allocation can be created if the current company doesn't have an employee linked to the loggedIn user.
```
The end of the comment doesn't seem necessary to explain what the test does